Assets/Scripts/Controllers/BuildModeController.cs

```diff
@@ -117,9 +117,9 @@ public void DoBuild(Tile t)
 
                 j.furniturePrototype = WorldController.Instance.World.furniturePrototypes[furnitureType];
 
-                for (int x_off = t.X; x_off < (t.X + WorldController.Instance.World.furniturePrototypes[furnitureType].Width); x_off++)
+                for (int x_off = t.X; x_off < (t.X + j.furniturePrototype.Width); x_off++)
```

Comment:
Please change name of "j" without scrolling to the top it's not possible to know what is "j"